tests/ClientServer.h

```diff
@@ -187,8 +187,9 @@ class ClientServer
     auto& getJsonRpcClient() { return _client; }
     void process()
     {
-        _wsClient.process(10);
-        _brayns->preRender();
+        _wsClient.process(CLIENT_PROCESS_TIMEOUT);
+        for (size_t i = 0; i < SERVER_PROCESS_RETRIES; ++i)
+            _brayns->preRender();
```

Comment:
Check the above, not exactly retires at the preRender is always executed n times